spec/presenters/root_browse_page_presenter_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe RootBrowsePagePresenter do
+
+  describe "#render_for_publishing_api" do
+    it "raises a RunTimeError if top-level browse pages are not present" do
+      expect {
+        RootBrowsePagePresenter.new.render_for_publishing_api
+      }.to raise_error(RuntimeError)
+    end
+
+    context "with top level browse pages" do
+
+      let!(:top_level_page_1) { create(:mainstream_browse_page)}
+      let!(:top_level_page_2) { create(:mainstream_browse_page)}
+
+      setup do
```

Comment:
Also not that this is rspec - the `setup` block is actually not run here. This masks a bug here:

https://github.com/alphagov/collections-publisher/pull/89/files#diff-9ceec5e0ecaaf0a70e37d3077c9b5b80R30